frontend/src/styles/global.scss

```diff
@@ -779,3 +779,27 @@ body {
         border-radius: var(--radius);
     }
 }
+
+.loading-overlay {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: $bg_charcoal;
+    text-align: center;
+    min-height: 6rem;
+    opacity: 0.7;
+    z-index: $z_content_overlay;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &.over-table {
+        display: block;
+        background: rgba(0, 0, 0, 0.1);
+        td {
+            display: block;
+        }
+    }
+}
```

Comment:
Why the move from `style.scss`?

Reply:
The file is listed as deprecated so I thought better not to edit